src/ZiraLink.Api.Application/Tools/Tools.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.NetworkInformation;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace ZiraLink.Api.Application.Tools
+{
+    public class Tools : ITools
+    {
+        public async Task<bool> CheckDomainExists(string domainUrl)
+        {
+            HttpClient _client = new HttpClient();
+            try
+            {
+                HttpResponseMessage response = await _client.GetAsync(domainUrl);
+                var bb = response.Content.ReadAsStringAsync();
+                return false;
+            }
+            catch (Exception)
+            {
+            }
+
+            try
```

Comment:
Same here, probably try-catch is not necessary here

Reply:
Same we need try-catch because when the url or domain doesn't find, it will get error on the catch.